relayer/crates/starknet-chain-components/src/impls/types/config.rs

```diff
@@ -17,4 +19,19 @@ pub struct StarknetChainConfig {
     pub relayer_wallet: StarknetWallet,
     #[serde(with = "humantime_serde")]
     pub poll_interval: Duration,
+    pub contract_addresses: StarknetContractAddresses,
+    pub contract_classes: StarknetContractClasses,
```

Comment:
We can probably refactor the code later to always use contract addresses instead of class hashes. I originally used class hashes, with the assumption that we may switch to permissionless IBC contracts on Starknet. But now that seems increasingly unlikely to happen.

Reply:
Ah. I wasn't aware. Thanks for pointing this out.